docs/umberto.json

```diff
@@ -121,13 +121,47 @@
 		"features/spell-checker.html": "features/spelling-and-grammar-checking.html",
 		"features/collaboration/comments/comments-api.html": "api/comments.html",
 		"features/ckbox.html": "features/images/image-upload/ckbox.html",
-		"framework/guides/ui/document-editor.html": "framework/guides/deep-dive/ui/document-editor.html",
```

Comment:
Are these replaced with new links? Because we should not remove them totally.

Reply:
Yes, I added new redirects and updated the old ones to avoid double redirects.

Reply:
FIne, thanks!